### ISSUE TITLE
Add touch support

### ISSUE THREAD

**Issue by acusti** [tleunen/react-number-editor#8]
This is actually totally trivial thanks to the `touch` option in your [react-clickdrag](https://github.com/tleunen/react-clickdrag) component.

**Comment by tleunen** [comment]
I guess it will work, but I'm not sure about the behavior though.. Is it really usable on touch?
I was initially thinking about having a classic number input (or range input) for mobile with touches...

**Comment by acusti** [comment]
I’d recommend you [give it a try](http://www.acusti.ca/primal-multiplication/). The only change I made was the one shown in the PR. It works really well, I think; however, it’s also pretty much impossible to discover (which is why I added info text to explain it; in fact, the issue of discoverability also exists for users using a cursor). That’s really a UX issue, though, so it may be best handled on a case by case basis by the particular implementation of the control.
